main/src/main/scala/sbt/internal/SettingGraph.scala

```diff
@@ -60,8 +63,7 @@ object Graph {
   // [info]   | +-baz
   // [info]   |
   // [info]   +-quux
-  def toAscii[A](top: A, children: A => Seq[A], display: A => String): String = {
-    val defaultWidth = 40
+  def toAscii[A](top: A, children: A => Seq[A], display: A => String, defaultWidth: Int): String = {
     // TODO: Fix JLine
     val maxColumn = math.max( /*JLine.usingTerminal(_.getWidth)*/ 0, defaultWidth) - 8
```

Comment:
@eed3si9n what exactly has to be done here for the JLine? I see this code is used in 0.13 branch, and it seems to work well there (in my limited testing).